index.html

```diff
@@ -428,6 +429,22 @@ <h2>Glossary</h2>
 </section>
 
 
+<section class="appendix" id="how-to-use">
+	<h3 id="howto">How to use this glossary</h3>
+	
+	<p>If you are writing a W3C specification, you can use the terminology in this document directly by importing this document as a cross-reference source. This may be especially helpful when using <a>character encoding</a> or <a>locale</a>-related terms found here, since these terms are common to the needs of many specifications, are often "inter-locking" with other definitions, but might not otherwise provide value to your spec.</p>
+	
+	<p>If you are using <a href="https://respec.org">ReSpec</a>, you can import this glossary using the <kbd><a href="https://respec.org/docs/#xref">xref</a></kbd> keyword in your configuration block like this:</p>
+	
+<pre>
+	xref: [ "i18n-glossary" ],
+</pre>
+
+    <p>This allows you to then write references using the normal ReSpec notation (<code>&lt;a&gt;</code> tags or <code>[&#x200b;= term =&#x200b;]</code> markup).</p>
```

Comment:
I'm not sure if adding spaces at both ends of the term when using the `[=term=]` notation will work properly, but all the examples I've seen have no spaces, including their [official docs](https://respec.org/docs/#linking-to-concepts).

Reply:
Done.